src/main/java/io/omnition/loadgenerator/util/TraceGenerator.java

```diff
@@ -69,7 +67,10 @@ private Span createSpanForServiceRouteCall(Map<String, Object> parentTags, Servi
                 }
             }
         }
-        tagsToSet.put("instance", service.instanceName);
+        if (serviceTier.instances.size() > 0) {
+            String instanceName = serviceTier.instances.get(random.nextInt(serviceTier.instances.size()));
+            tagsToSet.put("instance", instanceName);
```

Comment:
how does this work? doesn't it need to be k8s.io/pod/name?

Reply:
removed.